test/helpers.py

```diff
@@ -242,20 +242,48 @@
             [1.0, 0.0, 1.0],
             [1.0, 1.0, 1.0],
             [0.0, 1.0, 1.0],
-            [0.0, 0.0, 2.0],
-            [1.0, 0.0, 2.0],
-            [1.0, 1.0, 2.0],
-            [0.0, 1.0, 2.0],
         ]
     ),
     [  # Split the lower cube into tets and pyramids. The upper cube is hexahedron
```

Comment:
This comment is no longer correct, is it?

Reply:
Corrected.